public/public-functions.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * The public-specific functionality of the plugin.
+ *
+ * @link       https://nataliesmith.dev
+ * @since      1.0.0
+ * @package    Poetwp
+ * @subpackage Poetwp/admin
+ */
+
+/**
+ * Enqueue public styles for the plugin.
+ *
+ * @return void
+ */
+function poetwp_enqueue_public_styles() {
```

Comment:
This type could go into the function signature itself

```suggestion
 */
function poetwp_enqueue_public_styles(): void {
```

Reply:
Nice one, I haven't seen this before.

My other question is - do you have to specify return void for functions that don't return a value?

Reply:
You don't _have_ to, but it's a good practice to, to confirm that it wasn't just that a type declaration was forgotten to be added.

https://www.php.net/manual/en/language.types.void.php is the note of when it was added (PHP 7.1.0).